agent/tool-scripts/datalog/openshift-metrics-datalog

```diff
@@ -11,34 +11,16 @@ interval = int(sys.argv[2])
 master = sys.argv[1]
 os.system('scp "%s:/etc/origin/master/admin.crt" "/run/pbench/keys/admin.crt"' % (master) )
 os.system('scp "%s:/etc/origin/master/admin.key" "/run/pbench/keys/admin.key"' % (master) )
-counter_regexp = re.compile(".*code=\"(\d+)\".*resource=\"(\S+)\",verb=\"(\S+)\"} (\d+)")
 while True:
   time_stamp = int(time.time())
   next_start = time_stamp + interval
   r = requests.get('https://%s:8443/metrics' %(master), cert=('/run/pbench/keys/admin.crt', '/run/pbench/keys/admin.key'), verify=False)
```

Comment:
Just a reminder that as was pointed out in IRC, the [`prom2json` tool](https://github.com/prometheus/prom2json) should probably be used by this tool to fetch the data as a JSON document, and then pull that JSON document in and adorn it with the proper timestamp.